sanic_jinja2/__init__.py

```diff
@@ -28,8 +75,14 @@ def init_app(self, app, loader=None, pkg_name=None, pkg_path=None):
         if not hasattr(app, 'extensions'):
             app.extensions = {}
 
+        if self.context_processors:
+            if not hasattr(app, CONTEXT_PROCESSORS):
+                setattr(app, CONTEXT_PROCESSORS, self.context_processors)
+                app.request_middleware.append(self.context_processors)
+
         app.extensions['jinja2'] = self
-        app.jinja_env = self.env
+        app.jinja2 = self.env
```

Comment:
this will break the code that uses the old version. If you want to change the name, please pass it via the argument.